File: gcr-codebase/programming-elements/calculatesimpleinterest.java
import java.util.Scanner;

public class calculatesimpleinterest {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int p = sc.nextInt(); //enter the principal amount
        int r = sc.nextInt(); //enter the rate of interest
        int t = sc.nextInt(); //enter the time
        float SI = (p*r*t)/100; //formula for calculating simple interest
        System.out.println(SI);
        sc.close();;
    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/CalculateSimpleInterest.cs
using System;
class CalculateSimpleInterest{
    static void Main()
    {
        int principal = int.Parse(Console.ReadLine()); //Taking principal amount as input
        float rate = float.Parse(Console.ReadLine()); //Taking Rate of interest as input
        int time = int.Parse(Console.ReadLine()); //Taking Time as input
        float SI = (principal * rate * time)/100.0f; //Calculating The Simple Interest.
        Console.WriteLine(SI); //Printing results.
    }
}